src/containers/ProfileEditForm/ProfileEditForm.validations.js

```diff
@@ -1,33 +1,38 @@
 export default {
```

Comment:
Ditto above: diff in this file is a bit messy, because I organised the keys alphabetically